description: Remove the "--disable-setuid-sandbox" command line flag as a bad flag

--- a/chrome/browser/ui/startup/bad_flags_prompt.cc
+++ b/chrome/browser/ui/startup/bad_flags_prompt.cc
@@ -43,7 +43,7 @@ void ShowBadFlagsPrompt(Browser* browser) {
     // These flags disable sandbox-related security.
     switches::kDisableGpuSandbox,
     switches::kDisableSeccompFilterSandbox,
-    switches::kDisableSetuidSandbox,
+    //switches::kDisableSetuidSandbox,
     switches::kDisableWebSecurity,
 #if !defined(DISABLE_NACL)
     switches::kNaClDangerousNoSandboxNonSfi,
